### PR TITLE
Add Kommun model and add to companies search (replace city) - #140161159

### PR DESCRIPTION
**Pull request by patmbolger**
PT Story: https://www.pivotaltracker.com/story/show/140161159

Changes proposed in this pull request:
1. Added model Kommun, and rake task to populate DB table
2. Changed Address to drop `kommun` and instead use `belongs_to` association
3. Modified views accordingly
4. Modified tests accordingly

Screenshots (Optional):
<hr>
Editing a company:
screen shot 2017-03-06 at 9 53 55 am

<hr>
Company search:
screen shot 2017-03-06 at 8 57 52 am

Ready for review:
@weedySeaDragon 
@thesuss

### CODE REVIEW

**Review by thesuss — APPROVED** [review]

**Comment by thesuss** [comment]
But check what this is.. 
whats that

**Comment by patmbolger** [comment]
@thesuss - that was in the code before.  Looks like a debugging entry - will fix in this PR.

**Comment by patmbolger** [comment]
Fixed errant "0" in company form.  Ready for review.

File: db/seeds.rb [hunk 1]
```diff
@@ -78,12 +78,14 @@ class SeedAdminENVError < StandardError
     puts "Run task 'shf:load_regions' before seeding if you want records created for"
     puts 'users, members, membership_applications, business categories and companies.'
   else
+    kommuns = Kommun.all.to_a
 
-    puts '\nSeeding the db with users...'
+    puts 'Seeding the db with users...'
```
Comment (weedySeaDragon) [inline db/seeds.rb:83]: glad you fixed that.

**Review by weedySeaDragon — APPROVED** [review]
Looks all good.

Just as an aside: it seems to me that we should add a method to Company to get the 'primary' address (which will just be == `addresses.first` for now).  That should be done in a different PR.  Seeing `company.addresses.first` so many places now makes it clear to me that we need to deal with that.

**Pull request merged**